packages/node/src/utils/transform-to-rollup-output.ts

```diff
@@ -69,12 +69,15 @@ function transformToRollupOutputAsset(asset: OutputAsset): RollupOutputAsset {
 export function transformToRollupOutput(output: Outputs): RollupOutput {
   const { chunks, assets } = output
 
+  if (chunks.length === 0) {
```

Comment:
I'm not sure how we get to here, there will be at least one entry chunk to be generated. I tend not to do this runtime check so far.